project/SeqexecEngineModules.scala

```diff
@@ -29,10 +29,6 @@ trait SeqexecEngineModules {
     .settings(
       libraryDependencies ++= Seq(BooPickle.value) ++ TestLibs.value
     )
-    .jsSettings(
-      scalaJSUseRhino := false,
```

Comment:
I thought Rhino was dead long time ago 😄, but TIL it's still quite active. Is scalajs using node by default now?